apps/portal-front/src/components/service/document/shareable-resource-slug.tsx

```diff
@@ -35,6 +37,18 @@ interface ShareableResourceSlugProps {
   updateActions?: ReactNode;
 }
 
+export enum ShareableResourceType {
+  OPENAEV_SCENARIO = 'openaev_scenario',
+  OPENCTI_CSVFEED = 'csv_feed',
+  OPENCTI_CUSTOMDASHBOARD = 'custom_dashboard',
+}
+
+export const SHAREABLE_RESOURCE_TYPE = {
```

Comment:
I think we could rename this constant to `SHAREABLE_RESOURCE_TYPE_NAME_MAPPING` or something like that to be clear about the content of this constant: displayed name for each resource type.